homeassistant/components/philips_js/__init__.py

```diff
@@ -187,3 +192,5 @@ async def _async_update_data(self):
             pass
         except AutenticationFailure as exception:
             raise ConfigEntryAuthFailed(str(exception)) from exception
+        except GeneralFailure as exception:
```

Comment:
Wouldn't `ConnectionFailure` above also be a reason to raise an `UpdateFailed`?

Reply:
Nah. That is handled differently. In that case the entities may be turned on by linked device trigger. So they should not be fully unavailable.